backend/module/authn/authn.go

```diff
@@ -104,3 +109,32 @@ func (a *api) Callback(ctx context.Context, request *authnv1.CallbackRequest) (*
 		AccessToken: token.AccessToken,
 	}, nil
 }
+
+func (a *api) CreateToken(ctx context.Context, request *authnv1.CreateTokenRequest) (*authnv1.CreateTokenResponse, error) {
+	var prefixedSubject string
+	switch request.TokenType {
+	case authnv1.CreateTokenRequest_SERVICE:
+		prefixedSubject = "service:" + request.Subject
+	default:
+		return nil, status.Error(codes.InvalidArgument, "invalid token type")
+	}
```

Comment:
let's put this logic in the service, not the module, since that's where helpers live to deal with deciphering and understanding claims, subjects, etc, and we may want a helper that is service aware. plumb the token type one level deeper.